logger.js

```diff
@@ -154,7 +154,7 @@ function pinoLogger (opts, stream) {
     if (!res.log) {
       res.log = responseLogger
     }
-    if (!res.allLogs) {
+    if (!Array.isArray(res.allLogs)) {
```

Comment:
The prior check was a coercive check because it was against a truthy/falsy value. I'd prefer it if we make the new check explicit:

```suggestion
    if (Array.isArray(res.allLogs) === false) {
```

Reply:
makes sense to me, fixed